src/doc/dev_manual/pr_create.rst

```diff
@@ -0,0 +1,106 @@
+Creating a Pull Request
+=======================
+
+Overview
+--------
+
+The Pull Request (PR) is a mechanism that allows other developers on the team to review your work before merging your branch into develop. It is extremely useful for preventing bugs, enforcing coding practices, and ensuring changes are consistent with VisIt's overall vision. Although a review does slow down the developmenet process, there are several policies we put into place to ensure a review is completed in a timely manner, and the benefits of a review outweigh the added time.
```

Comment:
As I mentioned in telecon yesterday...I started doing a revision to clean up wording a bit and got pretty far and then Safari died. Do you mind if I edit this content a bit and commit a change on this branch?

Reply:
@markcmiller86 That's fine!